solr/solrj/src/test/org/apache/solr/client/solrj/embedded/AbstractEmbeddedSolrServerTestCase.java

```diff
@@ -104,4 +110,8 @@ protected SolrClient getSolrCore1() {
   protected SolrClient getSolrCore(String name) {
     return new EmbeddedSolrServer(cores, name);
   }
+
+  protected SolrClient getSolrAdmin() {
```

Comment:
Cool...    So in many of our tests we need an admin client that doesn't have a core/collection name in the path, and then a collection specific one, and this solves that first need?

Reply:
True. The client used by these tests do not need a core